languages.toml

```diff
@@ -2775,7 +2775,7 @@ source = { git = "https://github.com/kylegoetz/tree-sitter-unison", rev = "98c4e
 [[language]]
 name = "todotxt"
 scope = "text.todotxt"
-file-types = [{ suffix = ".todo.txt" }, "todotxt"]
+file-types = [{ glob = ".todo.txt" }, "todotxt"]
```

Comment:
I am not sure this is quite the same thing i believe the indention was to allow `*.todo.txt` although I never use that format so I am not sure

Reply:
@Jan9103 As you added todotxt, what do you think?

Reply:
Thanks for filling jn the details. It should be `[{ glob = "todo.txt" },  { glob = "*.todo.txt" }, "todotxt", "todo.txt"]` with the syntax from this PR. Plaintext now only matches file extensions with this PR, so you need a glob for the " only filename case". By default globs match relatively paths but if your glib starts with `/` or `*` it matches an absolute path instead